src/app.ts

```diff
@@ -61,18 +61,26 @@ app.on('second-instance', () => {
 })
 
 function createWindow(): void {
+  const lastWindowState = config.get('lastWindowState') as LastWindowState
```

Comment:
For some reason I need to assert the type here because
1. `lastWindowState` will be typed as `boolean | LastWindowState`
2. referencing `lastWindowState.bounds` will throw an error that `bounds` doesn't exist on `boolean`

It's a weird error and I don't know if it's by design, but we should check with the [`electron-store`](electron-store) guys.

This is done everywhere where `config.get()` is called.

Reply:
Okay, I think this is actually by design and you must assert the type.

Reply:
Rolling back to `electron-store@3.1.0` without type definitions and using `@types/electron-store` has fixed this behaviour and `config.get()` is now returning the correct type based on the key. I'll check the type definition in `electron-store` and see if I can contribute back.